src/integration-tests/bash/run.sh

```diff
@@ -1299,7 +1289,23 @@ function create_domain_pv_pvc_load_balancer {
       trace "helm install output:"
       cat $outfile
     else
-      fail "create-weblogic-domain.sh is longer supported"
```

Comment:
Are you sure that the code to call these sample scripts should be in this 'else' clause?

Reply:
So, USE_HELM now is used to say whether to setup the domain using the domain helm chart, or whether to setup the domain using the sample scripts?

I think it used to indicate whether to setup both the operator and domain with helm, v.s. the old create scripts.

Maybe it should be renamed to USE_DOMAIN_HELM_CHART?

Reply:
On the other hand, run.sh is supposed to be deleted soon so it's probably not worth the effort to rename it.

Reply:
We could rename the property, but I don't think USE_HELM is causing any confusion because we don't support setting up operator using scripts any more. So USE_HELM only means use helm for the domain creation or not. It would make more sense to rename the property (and add another property USE_OPERATOR_HELM_CHART) if we support both script and helm for operator, and support mixed cases between operator and domain.

Reply:
There are no plans to configure the operator with scripts - we only support helm.

Anyway, I wouldn't bother renaming the property since run.sh will go away soon.